ingestion_tools/scripts/common/yaml_files.py

```diff
@@ -35,13 +36,18 @@ def get_yaml_config_files(
         if filename in EXCLUDE_LIST:
             continue
         if any(keyword in filename for keyword in exclude_keywords_list):
-            print(f"[INFO]: Excluding {file} because it contains an exclude keyword")
+            if verbose:
+                print(f"[INFO]: Excluding {file} because it contains an exclude keyword")
```

Comment:
nit: Typically we'd use the python logging library here (docs -- https://docs.python.org/3/library/logging.html). For example, we'd set the default log output level to `INFO` and every call to `logger.info()` would print a message to stdout. Output like the stuff above that we want to hide would probably be wrapped with `logger.debug()` and if the `verbose` flag is set, we'd tell the logging library to set the default log output level to `DEBUG`

Reply:
I think we address fix this before merging the code.